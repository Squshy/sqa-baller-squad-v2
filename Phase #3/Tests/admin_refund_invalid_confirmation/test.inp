LOGIN
ADMINBOY
AdPass123
REFUND
ABC123
AKG
logout
exit
