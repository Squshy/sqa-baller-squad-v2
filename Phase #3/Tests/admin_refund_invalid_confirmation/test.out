=========================================================
                        DELETE
=========================================================

Enter Transaction ID to refund: ABC123

Are you sure you want to refund this
transaction? (Yes/No): KAI

---------------------------------------------------------
                    Invalid Confirmation.
=========================================================






